app/views/users/show.html.erb

```diff
@@ -1,2 +1,49 @@
-<h1>List of all users by id</h1>
+<div class="card mb-3 text-center" style="max-width: 540px;">
+  <div class="row g-0 text-center">
+    <div class="col-md-4">
+      <img src="<%= @user.photo %>" class="img-fluid rounded-start" alt="profile">
+    </div>
+    <div class="col-md-8">
+      <div class="card-body">
+        <h4 class="card-title"><%= @user.name %></h4>
+        <p class="card-text"><h6>Number of posts: <%= @user.post_counter %></h6></p>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="card mb-3 border-info" style="width: 680px;">
+    <div class='card-header'>Bio</div>
+    <div class="card-body">
+      <p class="card-text"><%= @user.bio %></p>
+    </div>
+  </div>
+   </div>
+  <% if @user.recent_posts.empty? %>
+    <div class="card mb-3" style="width: 540px;">
+      <div class="card-header ">No Posts</div>
+      <div class="card-body">
+        <h5 class="card-title">Please submit a new post.</h5>
+      </div>
+    </div>
+  <% else %>
+    <% @user.recent_posts.each do |post| %>
+      <%= link_to user_post_path(user_id:@user.id, id: post.id), class:'nav-link' do%>
+        <div class="card mb-3" style="width: 540px;">
+          <div class="card-header"><%= post.title %></div>
+          <div class="card-body">
+            <h5 class="card-title"><%= post.text %></h5>
+          </div>
+          <div class='card-footer d-flex gap-2 justify-content-end'>
+            <p class="card-text"><span class="text-muted">Comments: <%= post.comment_counter %></span></p>
+            |
+            <p class="card-text"><span class="text-muted">Likes: <%= post.likes_counter %></span></p>
+          </div>
+        </div>
+      <% end %>
+    <% end %>
+  <% end %>
```

Comment:
- [ ] Kindly make the user show page layout to match [the required design](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_page.png?token=GHSAT0AAAAAACD3NNQDVD37K6DLBAQC4MPIZEIR6TQ). You can also see that the items are centered and stretched to cover enough space on the page.

  ![image](https://github.com/enis-memic/blog-app/assets/81468675/66fc0446-04ea-47d5-ad00-13de0043f387)